reports/evaluation/template-plot-forecasts.Rmd

```diff
@@ -0,0 +1,41 @@
+```{r prediction-plots, echo = FALSE, results='asis', fig.width = 6.5, fig.height=8}
+
+for (forecast_date in forecast_dates) {
```

Comment:
Can this be by (1-4 week) horizon ending on a given date, the date on which the report is created?